common/v2/components/GeneralStepper/GeneralStepper.tsx

```diff
@@ -12,13 +12,15 @@ export interface StepperProps {
   defaultBackPath?: string; // Path that component reverts to when user clicks "back" from first step in flow.
   defaultBackPathLabel?: string;
   completeBtnText?: string; // Text for btn to navigate out of flow in last step.
+  onRender?(goToNextStep: () => void): void;
```

Comment:
Can you explain what this is supposed to be?

Reply:
To use the `useEffect` hook along with the `goToNextStep` I needed to be able to access it on render. It's just a way of accessing the `goToNextStep` from outside the stepper component.